src/rdkafka_request.c

```diff
@@ -1964,7 +1964,8 @@ int rd_kafka_ProduceRequest (rd_kafka_broker_t *rkb, rd_kafka_toppar_t *rktp) {
         rd_dassert(cnt > 0);
 
         rd_atomic64_add(&rktp->rktp_c.tx_msgs, cnt);
-        rd_atomic64_add(&rktp->rktp_c.tx_bytes, MessageSetSize);
+        rd_atomic64_add(&rktp->rktp_c.tx_msg_bytes, MessageSetSize);
```

Comment:
this isn't consistent with `rx_msg_bytes`, which is `rkm->rkm_key_len + rkm->rkm_len`.